numpy/_core/arrayprint.py

```diff
@@ -1247,6 +1247,12 @@ def format_float_positional(x, precision=None, unique=True,
     -------
     rep : string
         The string representation of the floating point value
+    
+    Raises
+    ------
+    RuntimeError
+        If the decimal string representation of the floating point scalar is longer
+        than 16384 characters
```

Comment:
```suggestion
```
I'll just remove this, thanks.  But I still feel if someone sees it, we should change it and I don't think it is very informative overall.

(If anyone disgrees, happy to follow up.)